reactions/wot_reaction.rb

```diff
@@ -0,0 +1,46 @@
+module Reactions
+  class WotReaction
+    class << self
+      def attributes
+        {
+            contains: /wot in tarnation/i
+        }
+      end
+
+      def obtain_random_image()
```

Comment:
Methods without arguments leave out the parenthesis in ruby

```suggestion
      def obtain_random_image
```